modules/client/src/controllers/WithdrawalController.ts

```diff
@@ -38,7 +38,17 @@ export class WithdrawalController extends AbstractController {
           recipient,
         );
         this.log.info(`Withdraw Response: ${JSON.stringify(withdrawResponse, replaceBN, 2)}`);
-        const minTx = withdrawResponse.transaction;
+        const minTx: CFCoreTypes.MinimalTransaction = withdrawResponse.transaction;
+        // set the withdrawal tx in the store
+        // TODO: better way to set store path
+        await this.connext.store.set([
+          {
+            path: `CF_NODE:store/${this.cfCore.publicIdentifier}/latestNodeSubmittedWithdrawal`,
```

Comment:
In my PR #362 I moved the prefix into a constant in types. We should do that here too. Hopefully my PR gets merged first so you can piggyback.